src/docdb/tree/DocDBAccountTreeItemBase.ts

```diff
@@ -18,8 +18,8 @@ export abstract class DocDBAccountTreeItemBase extends DocDBTreeItemBase<Databas
     public readonly label: string;
     public readonly childTypeLabel: string = "Database";
 
-    constructor(id: string, label: string, documentEndpoint: string, masterKey: string) {
-        super(documentEndpoint, masterKey);
+    constructor(id: string, label: string, documentEndpoint: string, masterKey: string, isEmulator?: boolean) {
```

Comment:
Why is isEmulator optional?

Reply:
The `CosmosDBAccountProvider` class uses this constructor, and currently, I see no use of adding an `isEmulator` field there.